### PR TITLE
interceptor: rename ocinterceptor to octrace.

### PR DESCRIPTION
**Pull request by songy23**
Updates https://github.com/census-instrumentation/opencensus-service/issues/136.

Rename trace interceptor to differentiate from metrics interceptor.

### CODE REVIEW

File: interceptor/opencensustrace/doc.go [hunk 1]
```diff
@@ -12,6 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package ocinterceptor is the logic for receiving OpenCensus trace protobuf defined spans from
+// Package octraceinterceptor is the logic for receiving OpenCensus trace protobuf defined spans from
 // already instrumented applications and then passing them onto a spanreceiver instance.
-package ocinterceptor
+package octraceinterceptor
```
Comment (odeke-em) [inline interceptor/opencensustrace/doc.go]: Why don't we just name it `octrace`?

**Comment by odeke-em** [comment]
Thank you @songy23 for working on this! I just have one request, could we instead just name it `octrace`  or `opencensustrace` since it is already a trace interceptor and also preserve the picked package name?

File: interceptor/opencensustrace/doc.go (hunk 1 above)
Comment (songy23) [reply interceptor/opencensustrace/doc.go]: `octrace` sounds good, updated.

**Comment by songy23** [comment]
Thanks!

**Pull request merged**